services/app-web/src/common-code/data/statePrograms.json

```diff
@@ -156,6 +156,21 @@
                     "id": "7cec9b0c-3833-4933-bc05-a9287a76d3ec",
                     "fullName": "Health Homes",
                     "name": "HHP"
+                },
+                {
+                    "id": "",
```

Comment:
You need to generate UUIDs for the new programs and put them in the spreadsheet. I usually just use https://www.uuidgenerator.net